api/enterprise-api-ref/first-time-setup.markdown

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+title: Fist Time Setup API
+published: true
+---
+
+The First Time Setup API enables creation of the initial administrator user. This API is only accessible when the system hasn't been previously configured. Once setup is complete, this API becomes permanently inaccessible.
```

Comment:
here again? capitalization? I dunno... 

```
The First time setup API enables...
```

see https://docs.cfengine.com/docs/3.24/api-enterprise-api-ref.html, they all match this title case we mention in documentation README mentioned above.

Reply:
also one sentence per line is nice.
```suggestion
The First time setup API enables creation of the initial administrator user.
This API is only accessible when the system hasn't been previously configured.
Once setup is complete, this API becomes permanently inaccessible.
```

Reply:
If you run cf-hub --new-setup-code won't it not be "permanently" inaccessible aka made accessible again?

So maybe "Once setup is complete, this API becomes inaccessible until a new setup code is generated as described below."